src/components/InvoicesList/InvoicesList.js

```diff
@@ -99,8 +102,11 @@ const InvoicesList = ({ dependencies: { dopplerBillingApiClient } }) => {
                           <td>{invoice.accountId}</td>
                           <td>{invoice.product}</td>
                           <td>
-                            <FormattedDate value={invoice.date} />
+                            {!!invoice.creationDate && (
+                              <FormattedDate value={invoice.creationDate} />
+                            )}
                           </td>
+                          <td>{!!invoice.dueDate && <FormattedDate value={invoice.dueDate} />}</td>
```

Comment:
Que ocurre cuando `dueDate` es _falsy_?

Kent Doods sugiere usar siempre el ternario en estos casos: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
Era mi idea, el tema que si la fecha es nulla no tiene que hacer nada y en ese caso no sabia que hacer. Alguna sugerencia?

Reply:
Ahi vi que se puede setear esto: ```: null```

Reply:
Done